src/lib/organisms/methodsHeader.svelte

```diff
@@ -88,6 +88,21 @@
     font-weight: 100;
   }
 
+  .button-pdf {
+	  color: white;
+	  font-family: var(--vtPrimaryFont);
+	  background-color: var(--vtDarkBlue);
+    padding: 0.5em 0.4em ;
+	  border-radius: 0.2em;
+	  margin-left: 1em;
+    margin-bottom: 0.5em;
+	}
+  
+	:hover .button-pdf {
+	  color: var(--vtDarkBlue);
+	  background-color: rgb(188, 188, 188);
+	}
+  
```

Comment:
Fix indenting inconsistencies

Reply:
I don't have these inconsistencies in my visual studio code...

Reply:
Check, must be a GitHub thingy ... nevermind!